### PR TITLE
Staging voice vidio detect

### PR DESCRIPTION
**Pull request by chaitanyagodavarthi**

### CODE REVIEW

**Comment by blogic** [comment]
Hi,
the nf code is missing the license headers and the osynbc ones have plume and not spdx bsd heders.
   John

File: feeds/wlan-ap/opensync/patches/18-video-voice-detect.patch [hunk 1]
```diff
@@ -3638,7 +3638,6 @@
 +    struct sip_call_start *SipCallStart = NULL;
 +    struct sip_call_end *SipCallEnd = NULL;
 +    struct sip_call_report *SipCallReport = NULL;
-+    int i = 0;
```
Comment (blogic) [inline feeds/wlan-ap/opensync/patches/18-video-voice-detect.patch]: assuming that this fixes code you added prior in the PR it should be folded into the prior patch

Comment (chaitanyagodavarthi) [reply feeds/wlan-ap/opensync/patches/18-video-voice-detect.patch]: should I squash all the patches in to a single commit?

**Comment by blogic** [comment]
merged into pending, Thanks !

**Pull request closed without merging**